chef/cookbooks/barclamp/libraries/nic.rb

```diff
@@ -566,7 +566,6 @@ def add_slave(slave)
       usurp(slave)
```

Comment:
usurp() removes addresses, routes and flushes the device, which might be required if some interface was configured manually and there is no wicked configuration for it -- however, I can test this with the usurp() call removed, and see what happens.